autosklearn/ensembles/ensemble_selection.py

```diff
@@ -18,12 +19,13 @@
 class EnsembleSelection(AbstractEnsemble):
     def __init__(
         self,
-        ensemble_size: int,
         task_type: int,
         metrics: Sequence[Scorer] | Scorer,
+        random_state: Optional[Union[int, np.random.RandomState]],
+        backend: Backend,
```

Comment:
I guess the `backend` is for some sort of compatibility thing but just to clarify that it doesn't use it?

Reply:
Will update this.